tests/fixtures/README.md

```diff
@@ -18,6 +18,10 @@ The end of the file contains a GUIDed footer table with entries that hold the
 SEV-ES AP reset vector address and SNP metadata table, which are needed in
 order to compute VMSAs for SEV-ES guests and the list of SNP measured pages.
 
+`ovm_amd_sev_snp.fd` is a matching OVMF build from commmit [5a67a2e](https://github.com/AMDESE/ovmf/commit/5a67a2efa7f0542f5afc7eb70f75bc8f6a6c4d24).
```

Comment:
Small typo:

```suggestion
`ovmf_amd_sev_snp.fd` is a matching OVMF build from commmit [5a67a2e](https://github.com/AMDESE/ovmf/commit/5a67a2efa7f0542f5afc7eb70f75bc8f6a6c4d24).
```

Also, can you please add which flavor of OVMF was built? OvmfX64 or AmdSev?

I see I already gave up before on keeping the binaries small, so we might as well have the full OVMF.fd here...

Reply:
I was going to ask, I can take the last 4 KB as we did on the other test cases. And this is an AmdSev build, it's just the most recent version that adds the new section type. Would you prefer if I delete the previous AmdSev OVMF and replace it with this one?

Reply:
While I'm glad the new code in this PR didn't break the measurement of the old OVMFs, I don't see a reason to keep endless history of those. The new SVSM_CAA section is already in upstream OVMF code.

Therefore, I suggest:
1. Take only the last 4 KB of your new AmdSev build.
2. Replace the existing AmdSev file in test data.
3. Fix all the measurements in existing tests.
4. Remove the new test cases you added (they are no longer needed).
5. Do the same with the OvmfX64 build.

Does this sound like the correct path forward to you?